static/templates/invite_user.hbs

```diff
@@ -33,16 +33,18 @@
                     <label for="expires_in">{{t "Invitation expires after" }}</label>
                     <div>
                         <select id="expires_in">
-                            {{!-- <option name="expires_in" value="{{ expires_in_options.hour.value }}">{{t "1 hour"}}</option> --}}
-                            <option name="expires_in" value="{{ expires_in_options.day.value }}">{{t "1 day"}}</option>
-                            <option name="expires_in" value="{{ expires_in_options.threeDays.value }}" >{{t "3 days"}}</option>
-                            <option name="expires_in" value="{{  expires_in_options.tenDays.value }}" selected="selected" >{{t "10 days"}}</option>
-                            <option name="expires_in" value="{{ expires_in_options.thirtyDays.value }}">{{t "30 days"}}</option>
-                            {{!-- <option name="expires_in" value="{{ expires_in_options.never.value }}">{{t "Never expires"}}</option> --}}
+                            {{#each expires_in_options}}
+                                {{#if this.default}}
+                                <option selected name="expires_in" value="{{this.value}}">{{this.desciption}}</option>
+                                {{else}}
+                                <option name="expires_in" value="{{this.value}}">{{this.desciption}}</option>
+                                {{/if}}
```

Comment:
You can do this a bit more cleanly by just having the if statement inline, I think.

Reply:
By removing the else statement two occurrences of the option "10 days" will emerge. There might be a way to solve it by typing something in the lines of 
```html
<option selected="{{this.default}}" name="expires_in" value="{{this.value}}">{{this.desciption}}</option>
```
But not all browsers have support for this since the mere presence of "selected" indicates true. That is why I believe that it is the best practice to keep the if else statement.
If you know a better way to solve it please point me in the correct direction so that I may learn something new.

![image](https://user-images.githubusercontent.com/65904251/133958945-a97ae2ae-72f6-47a5-b9d2-6543c25572af.png)

Reply:
Oh yes I found a solution!
```html 
<option {{#if this.default }} selected {{/if}} name="expires_in" value="{{this.value}}">{{this.desciption}}</option>
```
I will commit this later today.

Reply:
That looks correct to me!